### DISCUSSION TITLE
[Moved to Issue #454]

### DISCUSSION THREAD

**Discussion by JJ00899** [noah-nuebling/mac-mouse-fix#453] [Other]
I have a Kensington Wireless Trackball (K75326WW)

I installed MMF and it was working fine. 5 minutes later, my mouse is dead. 
Will NOT work on any other computer.

Anyone else experience this? I need help as I am not trying to blow another $80 on a mouse.

**Comment by noah-nuebling** [comment]
Hi! Please refer to Issue #454 for any further discussion. That helps me keep an overview of all the problems and feature requests. Thanks!
